Misc/NEWS.d/next/Library/2018-03-12-19-58-25.bpo-33064.LO2KIY.rst

```diff
@@ -0,0 +1,6 @@
+lib2to3 now properly supports trailing commas after **kwargs in function
+signatures.
+
+# Write your Misc/NEWS entry below.  It should be a simple ReST paragraph. #
+Don't start with "- Issue #<n>: " or "- bpo-<n>: " or that sort of stuff.
+###########################################################################
```

Comment:
Some extraneous blurb stuff  to remove here.

Reply:
Duh! Fixed :)